sdk/python/tests/integration/offline_store/test_feature_logging.py

```diff
@@ -49,6 +47,17 @@ def test_feature_service_logging(environment, universal_data_sources, pass_as_pa
         ),
     )
 
+    store.apply(
```

Comment:
Same for all these changes: we need to prepare a detailed explanation for community reviewers